### DISCUSSION TITLE
How to make code snippets working inside of configuration files?

### DISCUSSION THREAD

**Discussion by Bottikowksi** [microsoft/vscode-discussions#916] [Extension Development QnA] [answered]
Hello together,

I want to create code snippets that I can use inside the 'latex.json' file (snippet file for latex). Therefore, I wrote some code inside the 'json.json' file. The problem is that the created code snippet (in 'json.json') cannot be used inside the 'latex.json' file. But if I create a file named 'example.json' (saved at the desktop) the code snippet (in 'json.json') works there. Does anyone know why this is the case? 

Thank you and best regards

**Comment by isidorn** [comment, marked as answer]
This sounds like a user question. I suggest to check out our docs https://code.visualstudio.com/docs/editor/userdefinedsnippets
Or to ask Bing Chat on that page for help. Thus closing this
